### PR TITLE
add Travis CI build

### PR DESCRIPTION
**Pull request by gidsg**
### Context
CI is good.

### Changes proposed in this pull request
add travis CI build

### Guidance to review
build should pass in Travis CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+language: python
+python: 3.7
+
+# command to install dependencies
+install: "make init"
+
+# command to run tests
+script:
+    - make lint
+    - make test
```
Comment (arnau) [inline .travis.yml]: We should have `make check` as well to ensure types are sound.

File: Makefile [hunk 2]
```diff
@@ -13,3 +13,8 @@ security_check:
 lint:
 	pipenv run flake8
 .PHONY: lint
+
+init:
```
Comment (arnau) [inline Makefile]: I'd rather stick to convention and use `install:` here.

Comment (arnau) [reply Makefile]: Actually, after thinking about it, I'm onboard with `init` for this task and use `install` for installing the CLI from source.

File: Makefile [hunk 3]
```diff
@@ -13,3 +13,8 @@ security_check:
...
+	pip3 install pipenv
```
Comment (arnau) [inline Makefile]: This is a bit problematic. First, you can install `pipenv` using other means (e.g. `brew`) and you might already have it.

Secondly, `pip3` implies you have Python 3 installed as `python3` and `pip3`. Which may not be the case. I would explain that the project depends on `pipenv` in the README (well, we'll do it anyway) so people can install it the way they think it's best for them.

Comment (gidsg) [reply Makefile]: this ceremony is that required by Travis CI for their base image. If you think this is confusing to other users we could rename it `travis-ci-init` or something?

Comment (arnau) [reply Makefile]: I would rather have it directly in `.travis.yml` then.

**Pull request merged**